libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -231,6 +250,7 @@ SRV_Channels::copy_radio_in_out(SRV_Channel::Aux_servo_function_t function, bool
             if (do_input_output) {
                 rc->read();
             }
+            set_output_unlimited_once(function);
```

Comment:
yeh, I am not sure about this one. For me output should be always protected, and I don't use rc radio so I lack of experience on this. But like Wickedshell do for deepstall or what is in plane servo lib, I can understand that the radio can be use for more extrem value.